src/common/graphcmn.cpp

```diff
@@ -944,16 +944,18 @@ wxGraphicsContext::CreateRadialGradientBrush(
 
 wxGraphicsFont wxGraphicsContext::CreateFont( const wxFont &font , const wxColour &col ) const
 {
-    return GetRenderer()->CreateFont(font,col);
+    wxDouble dpiX, dpiY;
+    GetDPI(&dpiX, &dpiY);
+    return GetRenderer()->CreateFont(font, col, wxSize(dpiX, dpiY));
```

Comment:
If we keep using `wxSize()`, I'd add `wxRound()` calls here, even if it doesn't change much in practice.